app/hooks/useModals.tsx

```diff
@@ -1,41 +1,37 @@
 import { useMantineTheme } from "@mantine/core"
-import {
-  openConfirmModal,
-  openModal,
-  useModals as MantineUseModals,
-} from "@mantine/modals"
+import { modals, useModals as useMantineModals } from "@mantine/modals"
 import { ModalSettings, OpenConfirmModal } from "@mantine/modals/lib/context"
 
 const useModals = () => {
   const theme = useMantineTheme()
-
-  const { modals } = MantineUseModals()
+  const { modals: openModals } = useMantineModals()
```

Comment:
If im reading this right, I would update this name. I believe this is a count of how many modals are open, so many change it to modalsOpen because the current name feels like an action.